packages/page-treasury/src/Overview/ProposalCreate.tsx

```diff
@@ -4,6 +4,7 @@
 import BN from 'bn.js';
 import React, { useMemo, useState } from 'react';
 
+import { permillOf } from '@polkadot/app-bounties/helpers';
```

Comment:
Unless absolutely needed, we should not import cross-pages.. So would revert this and keep it bounty-specific.